multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/steps/CreateOrUpdateAppStep.java

```diff
@@ -179,6 +184,13 @@ public void printStepEndMessage() {
             getStepLogger().debug(Messages.APP_CREATED, app.getName());
         }
 
+        private void addMetadataAutoscalerLabel() {
+            if (shouldApplyIncrementalInstancesUpdate(context)) {
```

Comment:
this method will only return true if we're in the second phase of the blue-green deployment. Apps are created in the first.
Create a new method that will not check the phase but only the parameter